.travis.yml

```diff
@@ -1,16 +1,29 @@
 sudo: false
-git:
-  depth: 2
-branches:
-  except:
-    - /^v\d/
+dist: trusty
 language: node_js
-node_js:
-  - iojs
-  - node
-notifications:
-  email: false
+
+env:
+  matrix:
+    - NODE_VERSION="8" NPM_VERSION="5"
+    - NODE_VERSION="8" NPM_VERSION="6"
+    - NODE_VERSION="9" NPM_VERSION="5"
```

Comment:
Node.js 9 is no longer maintained. https://github.com/nodejs/Release/commit/2ec87a51ebc6eef442fdaff1b6f5907a247c5416